tests/modules/nf-core/graphtyper/genotype/nextflow.config

```
@@ -0,0 +1,12 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    
+    withName: GRAPHTYPER_GENOTYPE_REGION {
+        ext.args = '--region MT192765.1'
+    }
+
+    withName: SAMTOOLS_VIEW {
+        ext.args = '--output-fmt cram'
```

```suggestion
        publishDir = { enabled:false }
        ext.args = '--output-fmt cram'
```
You better recreate your test.yml file because of this change. It disables the output of samtools/view, which doesn't need to be checked here :)

---

This change causes the samtools part of the pipeline (needed to convert bam to cram) to fail. Does it also make files unavailable to the pipeline?

---

This just states that the files created by this process don't need to be emitted to the output folder (but they can still be used by other processes). So it won't make any difference except for what files are in the test.yml

---

When I make that change, I get this error: 

```
INFO     Running 'graphtyper/genotype' test with command:                                                                                                                                     
         nextflow run ./tests/modules/nf-core/graphtyper/genotype -entry test_graphtyper_genotype_multi -c ./tests/config/nextflow.config -c                                                  
         ./tests/modules/nf-core/graphtyper/genotype/nextflow.config --outdir /tmp/tmpcjhe3pic -work-dir /tmp/tmp6hm3gll5                                                                     

CRITICAL Error running test workflow (exit code 1)                                                                                                                                            
         N E X T F L O W  ~  version 22.10.4                                                                                                                                                  
         Launching `./tests/modules/nf-core/graphtyper/genotype/main.nf` [sleepy_ritchie] DSL2 - revision: cbe4076280                                                                         
         WARN: A process with name 'GRAPHTYPER_GENOTYPE_REGION' is defined more than once in module script:                                                                                   
         /media/fosterz/external_ssd_2/files/projects/public/nextflow/modules/tests/modules/nf-core/graphtyper/genotype/main.nf -- Make sure to not define the same function as process       
         [-        ] process > test_graphtyper_genotype_multi:SAMTOOLS_VIEW       -                                                                                                           
         [-        ] process > test_graphtyper_genotype_multi:SAMTOOLS_INDEX      -                                                                                                           
         [-        ] process > test_graphtyper_genotype_multi:GRAPHTYPER_GENOTYPE -                                                                                                           
                                                                                                                                                                                              
         executor >  local (2)                                                                                                                                                                
         [08/d41870] process > test_graphtyper_genotype_multi:SAMTOOLS_VIEW (test1) [  0%] 0 of 2                                                                                             
         [-        ] process > test_graphtyper_genotype_multi:SAMTOOLS_INDEX        -                                                                                                         
         [-        ] process > test_graphtyper_genotype_multi:GRAPHTYPER_GENOTYPE   -                                                                                                         
                                                                                                                                                                                              
         executor >  local (2)                                                                                                                                                                
         [08/d41870] process > test_graphtyper_genotype_multi:SAMTOOLS_VIEW (test1) [  0%] 0 of 2                                                                                             
         [-        ] process > test_graphtyper_genotype_multi:SAMTOOLS_INDEX        -                                                                                                         
         [-        ] process > test_graphtyper_genotype_multi:GRAPHTYPER_GENOTYPE   -                                                                                                         
                                                                                                                                                                                              
         executor >  local (3)                                                                                                                                                                
         [08/d41870] process > test_graphtyper_genotype_multi:SAMTOOLS_VIEW (test1)  [  0%] 0 of 2                                                                                            
         [73/6959e8] process > test_graphtyper_genotype_multi:SAMTOOLS_INDEX (test1) [  0%] 0 of 1                                                                                            
         [-        ] process > test_graphtyper_genotype_multi:GRAPHTYPER_GENOTYPE    -                                                                                                        
                                                                                                                                                                                              
         executor >  local (3)                                                                                                                                                                
         [08/d41870] process > test_graphtyper_genotype_multi:SAMTOOLS_VIEW (test1)  [  0%] 0 of 2                                                                                            
         [73/6959e8] process > test_graphtyper_genotype_multi:SAMTOOLS_INDEX (test1) [  0%] 0 of 1                                                                                            
         [-        ] process > test_graphtyper_genotype_multi:GRAPHTYPER_GENOTYPE    -                                                                                                        
         Unknown method invocation `toAbsolutePath` on Boolean type                                                                                                                           
                                                                                                                                                                                              
                                                                                                                                                                                              
         WARN: Killing running tasks (2)                                                                                                                                                      
                                                                                                                                                                                              
         executor >  local (3)                                                                                                                                                                
         [fc/c906c1] process > test_graphtyper_genotype_multi:SAMTOOLS_VIEW (test2)  [  0%] 0 of 1                                                                                            
         [-        ] process > test_graphtyper_genotype_multi:SAMTOOLS_INDEX (test1) -                                                                                                        
         [-        ] process > test_graphtyper_genotype_multi:GRAPHTYPER_GENOTYPE    -                                                                                                        
         Unknown method invocation `toAbsolutePath` on Boolean type
```

Any idea why that would occur? The error message "Unknown method invocation `toAbsolutePath` on Boolean type" makes me think it has something to do with file paths

---

Ah my bad it should be with `[]` instead of `{}`

---

Ok, glad it was an easy fix! Its working now. I will update it tomorrow. Thanks!